tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -216,12 +213,13 @@
       }
       #graph-container {
         position: relative;
-        height: 100%;
+        height: 85%;
       }
       #graph {
+        position: relative;
         display: block;
         width: 100%;
-        height: 400px;
+        height: 100%;
```

Comment:
Hmm, what does this height adjustment change?

Reply:
This is actually borrowed  from your PR https://github.com/tensorflow/tensorboard/pull/895/files :)
This improves (but doesn't fully fix) the bug where the height of the graph board is only a fraction of its container element.